src/components/Login.jsx

```diff
@@ -1,31 +1,71 @@
 import PropTypes from 'prop-types'
 import React from 'react'
+import { withRouter } from 'react-router'
+
 import { isClient } from '../lib'
 import UserEdit from '../containers/UserEdit'
 
-const Login = ({ location }) => (
-  <div>
-    {isClient() ? '' /*window.AuthService.login(location.query.nextUrl)*/ : ''}
-    <form action="/login-callback" method="POST">
-      <input type="hidden" name="nextUrl" value={location.query.nextUrl} />
-      <div>
-        <label>email:</label>
-        <input type="text" name="email"/><br/>
-      </div>
-      <div>
-	<label>Password:</label>
-	<input type="password" name="password"/>
-      </div>
+class Login extends React.Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      active: 'login'
+    }
+  }
+
+  handleClick = e => {
+    this.setState({ active: e.target.name })
+  }
+
+  render() {
+    const auth0Strategy =
+      window.PASSPORT_STRATEGY === 'auth0' ||
+      window.PASSPORT_STRATEGY === ''
+    const auth0Login = isClient() && auth0Strategy
+
+    const { location, router } = this.props
+
+    return (
       <div>
-	<input type="submit" value="Submit"/>
-      </div>
-    </form>
-    <UserEdit  />
-  </div>
-)
+        {/* Use auth0 */}
+        {auth0Login && window.AuthService.login(location.query.nextUrl)}
+
+        {/* Show UserEdit component configured for login / signup */}
+        <section>
```

Comment:
I think this part will also have to be hidden if auth0Login is enabled.  My PR was naively replacing it.

Reply:
Yup! The conditional check on line 53 should cover this whole section. Good catch!